### PR TITLE
Remove Healthcheck

### PR DESCRIPTION
**Pull request by autophagy**
@chaudum Do you know of a straightforward way of testing the usecase with SSL enabled? As far as I know, to get SSL working on the docker container would require much more configuration than just enabling SSL. Thoughts?

### CODE REVIEW

**Comment by chaudum** [comment]
Testing HTTPS is not necessarily required IMO; we should be able to rely on `--insecure` to work as expected.

File: Dockerfile [hunk 1]
```diff
@@ -56,7 +56,7 @@ ENV CRATE_HEAP_SIZE 512M
 # if we cannot get any response from the CrateDB (connection refused, timeout
 # etc). If any response is received (regardless of http status code) we
 # consider the node as running.
-HEALTHCHECK --timeout=30s --interval=30s CMD curl --max-time 25 $(hostname):4200 || exit 1
+HEALTHCHECK --timeout=30s --interval=30s CMD curl --max-time 12 localhost:4200 || curl --max-time 12 https://localhost:4200 --insecure || exit 1
```
Comment (chaudum) [inline Dockerfile]: Do we need the healthcheck chain, or would it also work if the healthcheck always uses the `--insecure` flag?

Comment (autophagy) [reply Dockerfile]: I think we need the healthcheck chain - in an SSL enabled setup curling against `localhost:4200` even with ``--insecure`` returns an empty response from the server. Even if the node was working properly, the healthcheck would think it has failed.

**Comment by mfussenegger** [comment]
Could we instead of having a HTTP level check do a check on TCP level? E.g. with socat / nc?

**Comment by autophagy** [comment]
@mfussenegger something like ` printf "GET / HTTP/1.1\r\n\r\n" | nc localhost 4200 | grep HTTP/1.1 | awk {'print $2'}` ? Though, if our healthcheck gets complicated, I'd rather put it into a healthcheck.sh.

**Pull request merged**